hazelcast/src/test/java/com/hazelcast/internal/config/ConfigValidatorTest.java

```diff
@@ -269,7 +269,7 @@ public void test_checkNearCacheConfig_withPreLoaderConfig_onClients() {
         checkNearCacheConfig(MAP_NAME, nearCacheConfig, true);
     }
 
-    @Test(expected = IllegalArgumentException.class)
+    @Test
```

Comment:
This was the test which failed, when the `NearCachePreloaderConfig` was configured for a member -> needs to pass now.